sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.students.controller.ts

```diff
@@ -160,7 +160,11 @@ export class AssessmentStudentsController extends BaseController {
     return [
       ...pendingAndDeniedAppeals,
       ...studentInProgressAndDeclinedApplicationOfferingChangeRequests,
-    ];
+    ].sort(
+      (firstRequest, secondRequest) =>
```

Comment:
Just double checking, the order here is aligned with the "Assessment History" (the below table in the UI) order also, right?

Reply:
I did not follow @andrewsignori-aot . The assessment history API does not sort the objects as of now or I am missing something.

Reply:
While retrieving the `assessmentHistorySummary`, there is an order by `submittedDate` in descending order. So, yes "Assessment Request" is aligned with "Assessment History".